training/templatetags/trainingdata.py

```diff
@@ -20,7 +20,13 @@ def is_user_paid(user_obj):
       if invigilator:
         academic_id = user_obj.invigilator.academic_id
       else:
-        return False
+        student = Student.objects.filter(user=user_obj).exists()
+        if student:
+          student = Student.objects.filter(user=user_obj)[0]
```

Comment:
```suggestion
          list = Student.objects.filter(user=user_obj)
          student = list.exists()
          if student:
              student = list[0]
```
remove duplicate call of .filter(). assign Student.objects.filter(user=user_obj) to a variable.

alternatively, you may call .first(), if okay.

Reply:
Refactored & added new commit